src/cpp/src/image_generation/stable_diffusion_3_pipeline.hpp

```diff
@@ -244,44 +244,58 @@ class StableDiffusion3Pipeline : public DiffusionPipeline {
         size_t idx_hidden_state_2 = m_clip_text_encoder_2->get_config().num_hidden_layers + 1;
         ov::Tensor text_encoder_2_hidden_state = m_clip_text_encoder_2->get_output_tensor(idx_hidden_state_2);
 
-        // get positive prompt_2_embed_out
-        ov::Tensor pooled_prompt_embed_out, prompt_embed_out, pooled_prompt_2_embed_out, prompt_2_embed_out;
+        ov::Tensor text_encoder_3_output;
+        if (m_t5_text_encoder == nullptr) {
+            ov::Shape t5_prompt_embed_shape = {generation_config.num_images_per_prompt,
+                                               m_clip_text_encoder_1->get_config().max_position_embeddings,
+                                               transformer_config.joint_attention_dim};
+            std::vector<float> t5_prompt_embed(ov::shape_size(t5_prompt_embed_shape), 0.0f);
+
+            text_encoder_3_output = ov::Tensor(ov::element::f32, t5_prompt_embed_shape);
+            std::copy(t5_prompt_embed.begin(), t5_prompt_embed.end(), text_encoder_3_output.data<float>());
```

Comment:
we create `t5_prompt_embed` with zeros just to copy to `text_encoder_3_output`, let's fill `text_encoder_3_output` itself with:
```suggestion
            std::fill_n(text_encoder_3_output.data<float>(), text_encoder_3_output.get_size(), 0.0f);
```

Reply:
Done